r4/src/main/java/org/opencds/cqf/r4/providers/PlanDefinitionApplyProvider.java

```diff
@@ -150,6 +146,23 @@ private CarePlan resolveActions(Session session) {
             .collect(Collectors.toList());
     }
 
+    private void resolveAction(Session session, PlanDefinition.PlanDefinitionActionComponent action) {
+        // TODO - Apply input/output dataRequirements?
+        if (meetsConditions(session, action)) {
+            resolveDefinition(session, action);
+            resolveSubActions(session, action);
```

Comment:
I'm wondering if there's any negative implication to calling `resolveSubActions` before `resolveDynamicActions`. The `dynamicValue` on a `PlanDefinition.action`/`ActivityDefinition` allows for modifying the result from the earlier `resolveDefinition`. Not sure if the subActions could interfere somehow.
I'd suggest changing the order just to reduce the chance of any weirdness. (Not that we are using dynamic values anyway)

Reply:
As I think about this more, I can see it either way. We wouldn't want a dynamic value to stomp on the results from sub-actions, but at the same time might we want a dynamic value to modify something on a subaction? It's all academic anyway since we don't use subactions, so I'm thinking too much